src/Norway.js

```diff
@@ -87,38 +99,23 @@ class Norway extends React.Component {
   zoomOut() {
     const { width, height } = this.props;
     let view = this.makeViewBoxString(0, 0, width, height);
-    const svg = d3.select("#" + this.props.id);
+    const svg = this.getSVG();
     svg.classed("zoomed", false);
     svg.attr("viewBox", view);
     svg.selectAll(".municipality").style("pointer-events", "none");
     svg.selectAll(".county").style("pointer-events", "all");
```

Comment:
Pretty sure you can chain these, but it's been a long time since I've used d3 heavily